docs/k0s-in-docker.md

```diff
@@ -0,0 +1,31 @@
+# k0s in Docker
+
+We publish k0s container image for every release. By default we run both controller and worker in the same container to allow easy local test "cluster".
+
+You can run your own k0s-in-docker easily with:
+```sh
+docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.pkg.github.com/k0sproject/k0s/k0s:<version>
+```
+Just grab the kubeconfig with `docker exec k0s-controller cat /var/lib/k0s/pki/admin.conf` and paste e.g. into [Lens](https://github.com/lensapp/lens/).
+
+## Running workers
+
+If you want to attach multiple workers nodes into the cluster you can run separate containers for workers.
+
+First, we need a join token for the worker:
+```sh
+token=$(docker exec -t -i k0s k0s token create --role=worker)
```

Comment:
Command returns this error:
```
token=`docker exec 568503a4fb53 k0s token create --role-worker` 
2020-11-05 09:09:59.142734 I | flag provided but not defined: -role-worker
```

Reply:
You've got a typo, `--role-worker` vs. `--role=worker`

Reply:
I just realised that :D